### ISSUE TITLE
[REQUEST] WordPress <= 4.2.3 - wp_untrash_post_comments SQL Injection CVE-2015-2213

### ISSUE THREAD

**Issue by madmike33** [rapid7/metasploit-framework#6383]
https://github.com/WordPress/WordPress/commit/70128fe7605cb963a46815cf91b0a5934f70eff5

https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-2213

**Comment by h00die** [comment]
I may take a crack at this in the next few days/weeks.
http://blog.checkpoint.com/2015/08/11/finding-vulnerabilities-in-core-wordpress-a-bug-hunters-trilogy-part-ii-supremacy/ looks to be pretty good for details

**Comment by h00die** [comment]
Download at https://wordpress.org/download/release-archive/

**Comment by madmike33** [comment]
Thnx will be waiting :)

**Comment by madmike33** [comment]
@h00die any updates on this :)

**Comment by h00die** [comment]
After carefully re-reading the vulns, its not as simple as I thought.  A bunch of new WP functionality needs to be written or expanded (or import a gem).  I started doing that, wrote a bunch of the new functionality, and then the code disappeared.  I think I accidentally wrote it in a branch I deleted, or something of that nature.
Got disenfranchised losing all the work I put in, and haven't looked at it since.
